lib/venue.rb

```diff
@@ -1,2 +1,62 @@
 class Venue
-end
+    attr_reader :name, :capacity
+    attr_accessor :patrons
+  def initialize(name, capacity, patrons = [])
+    @name = name
+    @capacity = capacity
+    @patrons = patrons
+  end
+
+  # create method named add_patron
+  # takes a patron as an argument
+  # it puts that patron into the patrons array
+
+  # def patrons
+  #
+
+  def add_patron(patron)
+    [patron] << @patrons
```

Comment:
So the most simple way to add something to an empty array is this:
```
@patrons << patron
```
So you start with the array that you want to add to on the left and then you shovel the thing you want to add on the right